Framework/DataHandling/src/LoadRaw/isisraw.cpp

```diff
@@ -515,6 +516,7 @@ int ISISRAW::ioRAW(FILE *file, bool from_file, bool read_data) {
       }
     }
   }
+  // cppcheck-suppress uninitdata
   delete[] outbuff;
```

Comment:
This could also move to the else block I believe (provided the definition is also moved)